locations/spiders/bim_tr.py

```diff
@@ -0,0 +1,46 @@
+from scrapy import FormRequest, Spider
+
+from locations.categories import Categories, apply_category
+from locations.items import Feature
+from locations.pipelines.address_clean_up import clean_address
+
+
+class BimTRSpider(Spider):
+    name = "bim_tr"
+    item_attributes = {"brand": "BİM", "brand_wikidata": "Q1022075"}
+    start_urls = ["https://www.bim.com.tr/Categories/104/magazalar.aspx"]
+    no_refs = True
```

Comment:
Could you please map `item["addr_full"]` to `item["ref"]`? They seem pretty unique.
```suggestion
```